src/hooks/sdk/docs/useMeetingStatus.stories.mdx

```diff
@@ -8,6 +8,7 @@ The `useMeetingStatus` hook returns the meeting status from the enum below.
 enum MeetingStatus {
   Loading,
   Succeeded,
+  Reconnecting,
```

Comment:
Adding a new enum will break the builder's application with this ESLint option.
https://typescript-eslint.io/rules/switch-exhaustiveness-check

I wonder if we should consider it a breaking change. What do you think?

Reply:
Interesting, thanks for bringing it up. Definitely have to check on breaking change rules on enums in typescript.

Reply:
I could not find a rule for enums or more information on breaking changes on enums, but like you said Union type rule may apply, in that case it would be hard to extend the enum. I think, to avoid completely breaking, we can improve this in next versions where either we move to `type` or have `string` valued `enums` over `number` incremented one.

With this change for to be safe, I will add the enum at the end of enums values over doing it in middle. That will help us avoid breaking application if depending on index of enum.

Reply:
For example, if you add the `Reconnected` enum, the `else` case is no longer valid in a builder's application.

```ts
enum MeetingStatus {
  Failed,
  Succeeded,
}
 
function log(status: MeetingStatus) {
  if (status === MeetingStatus.Failed) {
    console.log('Failed!');
  } else {
    console.log('Everything is good');
  }
}
```

Regardless, adding an enum won't break the builder's application unless they have enabled `switch` linting rule.

Reply:
Right, but its very valid point of concern and two learnings from this:
1. Union types are preferred over enums generally in typescript.
2. If using enums, have string valued enums over indexed ones to have correct logical ordering in enums and have non-breaking changes if someone is using indexed enum.